bigframes/ml/llm.py

```diff
@@ -598,8 +598,8 @@ class TextEmbeddingGenerator(base.BaseEstimator):
         connection_name (str or None):
             Connection to connect with remote service. str of the format <PROJECT_NUMBER/PROJECT_ID>.<LOCATION>.<CONNECTION_ID>.
             If None, use default connection in session context.
-        row_limit (int or None):
-            The maximum number of rows that this model can process.
+        max_rows (int or None):
+            The maximum number of rows that this model can process per call.
```

Comment:
My feeling is this is better as a bigframes-wide setting. Alternatively, if we are just focusing on sem-join cardinality explosion, we can apply a narrow limit just for that case